protocols/generic_ngs_prep/README.md

```diff
@@ -0,0 +1,110 @@
+# Custom NGS Library Prep
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+## Categories
+* NGS Library Prep
+
```

Comment:
There should be a sub category here. I think that `* Generic` could be good, but up to you. Please see [this](https://raw.githubusercontent.com/Opentrons/Protocols/develop/protocols/6cd9d6/README.md) for an example

Reply:
bump

Reply:
I added "Generic" as the sub category, sounds good to me.